tests/kafkatest/version/version.py

```diff
@@ -1,17 +1,19 @@
-# Copyright 2015 Confluent Inc.
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I could have sworn rat would check these files. Are we missing the check on this file for some reason?